lib/index.js

```diff
@@ -75,4 +80,133 @@ Appboy.prototype.identify = function(identify) {
   // https://www.appboy.com/documentation/Web/#setting-user-ids
   if (!userId) return;
   window.appboy.changeUser(userId);
-};
\ No newline at end of file
+
+  var address = identify.address();
+  var avatar = identify.avatar();
+  var birthday = identify.birthday();
+  var email = identify.email();
+  var firstName = identify.firstName();
+  var gender = identify.gender();
+  var lastName = identify.lastName();
+  var phone = identify.phone();
+  var traits = clone(identify.traits());
+
+  window.appboy.getUser().setAvatarImageUrl(avatar);
+  window.appboy.getUser().setEmail(email);
+  window.appboy.getUser().setFirstName(firstName);
+  window.appboy.getUser().setGender(getGender(gender));
+  window.appboy.getUser().setLastName(lastName);
+  window.appboy.getUser().setPhoneNumber(phone);
+  if (address) {
+    window.appboy.getUser().setCountry(address.country);
```

Comment:
@ccnixon as far as I could tell, there were no consequences to passing undefined. the reason for the `if (address)` is because if there isn't an address object at all, then we get an error because it's trying to get `country` from something that doesn't exist.

appboy allows you to unset traits by passing null so I wanted to leave that as a functionality for users.